src/Octoshift/Commands/WaitForMigration/WaitForMigrationCommandHandler.cs

```diff
@@ -84,13 +85,15 @@ private async Task WaitForRepositoryMigration(string migrationId, GithubApi gith
             if (RepositoryMigrationStatus.IsSucceeded(state))
             {
                 _log.LogSuccess($"Migration {migrationId} succeeded for {repositoryName}");
+                _warningsCountLogger.LogWarningsCount(warningsCount);
```

Comment:
I quite like this pattern - it allows us to test the logging without setting expectations on the error message itself!